actions/websocket_actions.test.jsx

```diff
@@ -534,6 +550,12 @@ describe('reconnect', () => {
         reconnect(false);
         expect(syncPostsInChannel).toHaveBeenCalledWith('otherChannel', '12345');
     });
+
+    test('should call loadPluginsIfNecessary when socket reconnects', () => {
```

Comment:
Unfortunately I didn't come up with any solution other then splitting this test with the other two added. That is because reconnect isn't an async function, like `loadPluginsIfNecessary`.